src/flowcontrol/src/RSProcessManager.cpp

```diff
@@ -262,7 +262,7 @@ bool RSProcessManager::handleQuery(std::string &request, std::vector<std::string
 
 
     outInfo(FG_BLUE << "Executing Pipeline # generated by query");
-    visualizer_.setActiveAnnotators(newPipelineOrder);
+    // visualizer_.setActiveAnnotators(newPipelineOrder);
```

Comment:
this should stay...handling queries means executing only a subset of the annotators from the `fixedFlow`. We only want to scroll through the results of those.

Reply:
there is also a big block of commented code before these lines. Could you please delete them while your changing this line back:).

Reply:
I updated